### PR TITLE
refactor(molecules): SfCallToAction

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/805

# Scope of work
refactor of Call to action molecule to adjust to new design and update of font variables, 
sf-banner-light was modifier removed

# Screenshots of designs
image
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfCallToAction/SfCallToAction.stories.js [hunk 1]
```diff
@@ -1,27 +1,11 @@
...
-  optionsKnob as options
-} from "@storybook/addon-knobs";
+import { withKnobs, text, color } from "@storybook/addon-knobs";
 import SfCallToAction from "./SfCallToAction.vue";
 storiesOf("Molecules|CallToAction", module)
   .addDecorator(withKnobs)
   .add("Common", () => ({
     components: { SfCallToAction },
     props: {
-      customClass: {
```
Comment (aniamusial) [inline packages/vue/src/components/molecules/SfCallToAction/SfCallToAction.stories.js]: This cause an error in console - please remove the `customClass` prop assignment in the stories template :)

File: packages/shared/styles/components/molecules/SfCallToAction.scss [hunk 2]
```diff
@@ -2,64 +2,84 @@
...
-  background-color: var(--call-to-action-background, var(--c-black));
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: cover;
-  color: var(--call-to-action-color, var(--c-on-dark));
+  align-items: var(--call-to-action-align-items);
+  justify-content: var(--call-to-action-justify-content, left);
+  min-height: var(--call-to-action-min-height, 16.75rem);
+  padding: var(--call-to-action-padding, var(--spacer-base));
+  background-position: var(--call-to-action-background-position, center);
```
Comment (aniamusial) [inline packages/shared/styles/components/molecules/SfCallToAction.scss]: I think you can make a shortcut out of all background properties: https://developer.mozilla.org/pl/docs/Web/CSS/background :)

File: packages/shared/styles/components/molecules/SfCallToAction.scss [hunk 3]
```diff
@@ -2,64 +2,84 @@
...
-    margin: var(--call-to-action-button, var(--spacer-big) 0 0 0);
-    white-space: nowrap;
-  }
-  &__text-container {
-    margin: var(--call-to-action-text-container);
-  }
-  &--light {
-    --call-to-action-background: var(--c-light);
-    --call-to-action-color: var(--c-light);
+    height: var(--call-to-action-button-height, auto);
```
Comment (aniamusial) [inline packages/shared/styles/components/molecules/SfCallToAction.scss]: You shoudd overwrite the button classes: `--button-width` etc.

File: packages/shared/styles/components/molecules/SfCallToAction.scss (hunk 2 above)
Comment (AdamPawlinski) [reply packages/shared/styles/components/molecules/SfCallToAction.scss]: I can do it, but I prepared also the issue for creating mixin for this. https://github.com/DivanteLtd/storefront-ui/issues/865 
Should I change that or prepare mixin? What do you think @aniamusial @pspaczek ?

**Pull request merged**